packages/backend-storage/API.md

```diff
@@ -46,21 +46,24 @@ export type RoleAccessBuilder = {
 
 // @public (undocumented)
 export type StorageAccessBuilder = {
-    to: (actions: StorageAction[]) => StorageAccessDefinition;
+    to: (actions: StorageAction[]) => StoragePathAccessDefinition;
 };
 
 // @public (undocumented)
-export type StorageAccessDefinition = {
-    getResourceAccessAcceptor: (getInstanceProps: ConstructFactoryGetInstanceProps) => ResourceAccessAcceptor;
-    actions: StorageAction[];
-    ownerPlaceholderSubstitution: string;
-};
+export type StorageAccessRecord = Record<StoragePath, StoragePathAccessDefinition[]>;
 
 // @public (undocumented)
 export type StorageAction = 'read' | 'write' | 'delete';
 
 // @public
-export type StoragePrefix = `/${string}/*`;
+export type StoragePath = `/${string}/*`;

```

Comment:
what's that ? how does this work ?

Reply:
It's a [string template type](https://www.typescriptlang.org/docs/handbook/2/template-literal-types.html) that enforces the string must start with "/" and end with "/*"